movai_core_shared/logger.py

```diff
@@ -201,6 +191,16 @@ def _emit(self, record):
             "log_fields": syslog_fields,
         }
 
+        if asyncio._get_running_loop() is not None:
```

Comment:
this is a bug

asyncio.get_running_loop()¶
Return the running event loop in the current OS thread.

If there is no running event loop a RuntimeError is raised. This function can only be called from a coroutine or a callback.

Reply:
I use the internal function (_get_running_loop()) because it returns None instead of raising an Exception (It's much faster) 

it's not a bug